### PR TITLE
Fix/broken forum quote

### LINKED ISSUE #289: Quote on forum is broken
### How to reproduce
1. Go to a forum topic
2. Click on 'quote' button

### Expected behaviour
You go to the last page, scroll down and see that the quote is in the textbox

### Current behaviour
The text is in the textbox, but you are not navigated to the last page and scrolled down

### PR DESCRIPTION
**Pull request by Ellen-Wittingen**
fix #289

### CODE REVIEW

File: app/controllers/forum/categories/category/threads/thread/posts/index.js [hunk 1]
```diff
@@ -30,6 +31,24 @@ export default Controller.extend({
...
+          next(() => {
+            document
+              .getElementById('newForumPostCard')
+              .scrollIntoView({ behavior: 'smooth', block: 'center' });
+          });
+        });
+      } else {
+        document
+          .getElementById('newForumPostCard')
+          .scrollIntoView({ behavior: 'smooth', block: 'center' });
```
Comment (DrumsnChocolate) [inline app/controllers/forum/categories/category/threads/thread/posts/index.js:49]: duplicate code, please simplify if you can

File: app/controllers/forum/categories/category/threads/thread/posts/index.js [hunk 2]
```diff
@@ -30,6 +31,25 @@ export default Controller.extend({
     async newPostCreated() {
       await this.model.posts.reload();
     },
+    addquote(q) {
```
Comment (guidojw) [inline app/controllers/forum/categories/category/threads/thread/posts/index.js:34]: nit, but camelCase :)

**Review by guidojw — CHANGES_REQUESTED** [review]

**Comment by DrumsnChocolate** [comment]
oh shucks

**Comment by DrumsnChocolate** [comment]
I used intellij refactor first, but now I just did a find and replace in the project so I hope I didn't miss any now

**Review by guidojw — APPROVED** [review]

**Comment by DrumsnChocolate** [comment]
I didn't want to wait for the tests because the new changes from staging were completely unrelated

**Comment by guidojw** [comment]
I don't think this is working, the redirect and the scrolling

**Comment by Ellen-Wittingen** [comment]
It does work for me, on my computer and my phone (both Firefox), so that is weird.

**Comment by guidojw** [comment]
Hmm maybe my browser still had an old version cached, I'll try again

**Comment by guidojw** [comment]
Ah yes it's working, epic!

**Pull request merged**